### DISCUSSION TITLE
What's a good open source EDR solution for Security Onion?

### DISCUSSION THREAD

**Discussion by damienhull** [Security-Onion-Solutions/securityonion#4439] [General] [answered]
I know Wazuh works with Security Onion. Not sure that counts as an EDR solution. Is there a way to get something like Velociraptor to work with Security Onion?

Thanks!

**Comment by weslambert** [comment, marked as answer]
While we don't officially support it, I personally maintain an "at-your-own-risk" integration with Velociraptor here: https://github.com/weslambert/securityonion-velociraptor

**Reply by damienhull** [reply]
I was looking at Velociraptor and or setup. I think we're going to go with Wazuh. That has more of what we're looking for. 

Thanks!
